test/db/archos/linux-x64/dbg_drt

```diff
@@ -135,188 +135,188 @@ CMDS=<<EOF
 drt all~[0]
 EOF
 EXPECT=<<EOF
-ymm15
```

Comment:
same here. Do you know why?

Reply:
In this case in particular it seems even worse, as there are registers that seem completely out of place (e.g. `cwd` in between `r15w` and `r15d`). What's happening?

Reply:
That's because r15w, r15d and cwd are all at offset 0 in libr/anal/p/anal_x86_cs.c: line ~3519 and ~3577. That's the fourth column of the ESIL register file description.